src/main/scala/com/cloudera/sparkts/TimeSeriesRDD.scala

```diff
@@ -475,6 +475,35 @@ class TimeSeriesRDD[K](val index: DateTimeIndex, parent: RDD[(K, Vector)])
     ps.close()
   }
 
+  def saveAsParquetDataFrame(path: String, spark: SparkSession): Unit = {
+    // Write out contents
```

Comment:
Are you able to add some documentation that outlines:
* How the data is stored, e.g. row-wise vs. column-wise.
* How the data is readable.  Through timeSeriesRDDFromParquet?  Any other way?